pygments/lexers/arturo.py

```diff
@@ -0,0 +1,247 @@
+"""
+    pygments.lexers.arturo
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the Arturo language.
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import bygroups, default, DelegatingLexer, do_insertions,\
+    include, RegexLexer, this, using, words
+from pygments.token import Comment, Generic, Keyword, Name, Number, Operator,\
+    Other, Punctuation, String, Text
+
+from pygments.util import ClassNotFound, get_bool_opt
+
+__all__ = [
+    'ArturoLexer'
+]
+class ArturoLexer(RegexLexer):
+    """
+    For Arturo source code
```

Comment:
Could also add a link like in the PR description.

Reply:
Sure!